solver/src/http_solver.rs

```diff
@@ -141,6 +147,11 @@ impl HttpSolver {
                     balance1: amm.reserves.0,
                     balance2: amm.reserves.1,
                     fee: *amm.fee.numer() as f64 / *amm.fee.denom() as f64,
+                    // TODO: map uniswap fixed cost
+                    cost: CostModel {
+                        amount: 0,
+                        token: self.token_to_string(&amm.tokens.get().0),
```

Comment:
I believe here the cost should be in the native token.

Reply:
For the optimization model, it can be any token (that is part of the batch and connected to the main trading island).